test/benchmarks/material/table/basic-table.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Component, Input} from '@angular/core';
+
+/**
+ * Just a generic object whose keys are strings and whose values can be anything.
+ *
+ * This is used by BasicTable whose rows are objects whose keys correspond to the cols provided and
+ * whose values can be anything.
+ */
+export interface GenericObject {
```

Comment:
When I do this I get this error on line 26 of app.module.ts

`Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{}'.
  No index signature with a parameter of type 'string' was found on type '{}'.ts(7053)`

Reply:
Ah, I checked the TS utility types and found that what you actually want here is `Record<string, string>`:
https://www.typescriptlang.org/play/#code/MYewdgzgLgBAHgLhgJQKagE4BMA80MCWYA5gDQz5HEB8MAvDAN4C+A3AFDtwDaA5AIYAjYLwC69GLwDsvVkA